parseedn.el

```diff
@@ -27,6 +27,63 @@
 
 ;;; Code:
 
+;; The EDN spec is not clear about wether \u0123 and \o012 are supported in
+;; strings. They are described as character literals, but not as string escape
+;; codes. In practice all implementations support them (mostly with broken
+;; surrogate pair support), so we do the same. Sorry, emoji 🙁.
+;;
+;; Note that this is kind of broken, we don't correctly detect if \u or \o forms
+;; don't have the right forms.
+
+(require 'a)
+(require 'parseclj-lex)
+
+(defun parseedn--string (s)
+  ""
```

Comment:
Intentionally left this one blank. Not quite sure I understand everything going on here and I thought it was better to ask you for clarification. Maybe we could add some one-line comments to each of the `replace-regexp-string`s as well.